frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/RebaseCurrentBranchOntoParentAction.java

```diff
@@ -63,6 +67,8 @@ public void update(AnActionEvent anActionEvent) {
    */
   @Override
   public void actionPerformed(AnActionEvent anActionEvent) {
+    LOG.debug("Performing RebaseCurrentBranchOntoParent action");
```

Comment:
To make it more refactor-friendly, here an in other similar places:

```suggestion
    LOG.debug("Performing ${getClass().getSimpleName()} action");
```

Reply:
wow, where did it come from  xD

Reply:
Hmmm... `Performing RebaseCurrentBranchOntoParentAction action`? :smile: 
So probably even `LOG.debug("Performing ${getClass().getSimpleName()}");`